src/bridge.ts

```diff
@@ -958,24 +958,25 @@ export class Bridge {
      * Install a custom handler for an incoming HTTP API request. This allows
      * callers to add extra functionality, implement new APIs, etc...
      * @param opts Named options
+     * @param opts.authenticate Should the token be automatically checked. Defaults to true.
+     * @param opts.handler Function to handle requests
      * @param opts.method The HTTP method name.
      * @param opts.path Path to the endpoint.
-     * @param opts.checkToken Should the token be automatically checked. Defaults to true.
-     * @param opts.handler Function to handle requests
      * to this endpoint.
      */
     public addAppServicePath(opts: {
         method: "GET"|"PUT"|"POST"|"DELETE",
         path: string,
+        authenticate?: boolean,
         handler: (req: ExRequest, respose: ExResponse, next: NextFunction) => void,
     }): void {
         if (!this.appservice) {
             throw Error('Cannot call addAppServicePath before calling .run()');
         }
         const app: Application = this.appservice.expressApp;
-
+        const authenticate = opts.authenticate ?? true;
```

Comment:
```suggestion
        authenticate = opts.authenticate ?? true;
```

Is re-declaring a parameter on the top level of a function valid in JavaScript?
I'm getting `SyntaxError: Identifier 'hi' has already been declared` in a local test.

Is the linter dead?

Reply:
Should be fine? It's an opts object, not a parameter. (Which is why I can't do defaults)